### PR TITLE
Automatically mount internal partitions

### PR DESCRIPTION
**Pull request by orschiro**
I did not cover MATE because I don't know how it works with MATE.

### CODE REVIEW

File: configuration/mounting-paritions/en.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
++++
+title = "Automatically mounting internal partitions"
++++
```
Comment (JoshStrobl) [inline configuration/mounting-paritions/en.md]: You're using too many +'s. Should be 3.

File: configuration/mounting-paritions/en.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+title = "Automatically mounting internal partitions"
```
Comment (JoshStrobl) [inline configuration/mounting-paritions/en.md]: The words should be titalized.

File: configuration/mounting-paritions/en.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
++++
+# Automatically mounting internal partitions
```
Comment (JoshStrobl) [inline configuration/mounting-paritions/en.md]: The worlds should be titalized.

File: configuration/mounting-paritions/en.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+Solus can automatically mount your internal partitions on startup.
+
+## Budgie & GNOME
```
Comment (JoshStrobl) [inline configuration/mounting-paritions/en.md]: There needs to be dedicated sections for Budgie and GNOME, regardless of them being the same.

File: configuration/mounting-paritions/en.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+First, open the application Disks. Select the disk from the left you want to automatically mount.
```
Comment (JoshStrobl) [inline configuration/mounting-paritions/en.md]: All of these steps should using numbered lists. You should also accompany this with images.

**Comment by JoshStrobl** [comment]
Closing this PR, should be done on your own fork so image uploads I've requested are done in the appropriate directories.

File: configuration/mounting-paritions/en.md (hunk 1 above)
Comment (orschiro) [inline configuration/mounting-paritions/en.md]: I only see three.

image

Comment (JoshStrobl) [inline configuration/mounting-paritions/en.md]: Oh, yea, GitHub didn't differentiate between the Git addition mark and usual + in TOML lol.

**Pull request closed without merging**